docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -158,6 +158,19 @@ Dropped ticks are usually a sign of resource contention, specifically CPU conten
 
 *Further Reading*: xref:7.0@server:install:sizing-general[Sizing Guidelines]
 
+[#CB90067]
+=== Service DCP Rollback to Zero (CB90067)
+
+*Background*: A service has been forced to DCP rollback to zero.
+This may be undesirable because it means the service is attempting to resynchronize the entirety of the data set.
```

Comment:
This may be undesirable because it means -> This means

Reply:
Fixed :)